src/discussion-tool-selector/DiscussionToolSelector.scss

```
@@ -0,0 +1,16 @@
+.discussion-tool {
+    &:hover,
+    &:focus {
+        border-color: theme-color("primary", "hover") !important;
+    }
+}
+
+.features-table {
+    scroll-snap-type: x mandatory;
+
+    th {
+        min-width: 7.5rem;
```

Let's make it a bit larger. 
```suggestion
        min-width: 9rem;
```